spec/unit/pops/parser/parse_application_spec.rb

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/pops'
+
+# relative to this spec file (./) does not work as this file is loaded by rspec
+require File.join(File.dirname(__FILE__), '/parser_rspec_helper')
```

Comment:
I think _require_relative 'parser_rspec_helper'_ works OK. I tested it with both rake and rspec.